src/storage/sql_table.cpp

```diff
@@ -1,13 +1,37 @@
 #include "storage/sql_table.h"
 #include <set>
+#include <unordered_set>
+#include <utility>
 #include <vector>
+
 #include "common/macros.h"
 #include "storage/storage_util.h"
 
 namespace terrier::storage {
 
 SqlTable::SqlTable(BlockStore *const store, const catalog::Schema &schema, const catalog::table_oid_t oid)
     : block_store_(store), oid_(oid) {
+  UpdateSchema(schema);
+}
+
+SqlTable::~SqlTable() {
+  while (default_value_map_.CBegin() != default_value_map_.CEnd()) {
+    auto pair = *(default_value_map_.CBegin());
+    delete[] pair.second.first;
```

Comment:
For varlens that are not inlined, simply delete `pair.second.first` will leak memory.